selfdrive/car/toyota/interface.py

```diff
@@ -39,11 +39,12 @@ def get_params(candidate, fingerprint=gen_empty_fingerprint(), car_fw=[]):  # py
       ret.steerActuatorDelay = 0.3
 
     elif candidate == CAR.PRIUS_V:
+      stop_and_go = True
```

Comment:
does stock ACC have stop and go? if not, this should stay as it. shouldn't affect your smartDSU

Reply:
Stock ACC have stop and go in worldwide. But Toyota block the function in Taiwan.